packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx

```diff
@@ -93,6 +93,17 @@ class ColorPalette extends Component<ColorPaletteProps, ColorPaletteState> {
   componentWillUnmount() {
     this.removeEventListeners()
   }
+  get padding() {
```

Comment:
I add one `padding` property to shrink the area where the indicator can move, this will make the component look nicer